hudi-utilities/src/main/java/org/apache/hudi/utilities/schema/KafkaOffsetPostProcessor.java

```diff
@@ -62,6 +66,38 @@ public KafkaOffsetPostProcessor(TypedProperties props, JavaSparkContext jssc) {
   }
 
   @Override
+  public HoodieSchema processSchema(HoodieSchema schema) {
```

Comment:
This was not overridden on purpose in case a user is extending the deprecated method.